test/src/main/java/org/apache/accumulo/test/replication/GarbageCollectorCommunicatesWithTServersIT.java

```diff
@@ -196,18 +196,18 @@ public void testActiveWalPrecludesClosing() throws Exception {
 
     log.info("Writing a few mutations to the table");
 
-    BatchWriter bw = client.createBatchWriter(table);
+    try (BatchWriter bw = client.createBatchWriter(table)) {
+      byte[] empty = new byte[0];
+      for (int i = 0; i < 5; i++) {
+        Mutation m = new Mutation(Integer.toString(i));
+        m.put(empty, empty, empty);
+        bw.addMutation(m);
+      }
 
-    byte[] empty = new byte[0];
-    for (int i = 0; i < 5; i++) {
-      Mutation m = new Mutation(Integer.toString(i));
-      m.put(empty, empty, empty);
-      bw.addMutation(m);
+      log.info("Flushing mutations to the server");
+      bw.flush();
```

Comment:
I think flushing the batch writer could be dropped now that its closed. I think closing the batch writer will flush it.

Reply:
Included in the most recent commit.